internal/service/sagemaker/endpoint_configuration_test.go

```diff
@@ -775,6 +803,9 @@ resource "aws_sagemaker_endpoint_configuration" "test" {
     initial_instance_count = 2
     instance_type          = "ml.t2.medium"
     initial_variant_weight = 1
+    routing_config {
```

Comment:
I think the dedicated test is good enough, also this is not checked anywhere so lets revert this

Reply:
oh right, will do, forgot I committed this bit